### PR TITLE
Adding dirigible-che-editor-plugin

### PR DESCRIPTION
**Pull request by delchev**
…ins/org.eclipse.che.editor.dirigible/1.0.0/meta.yaml

### What does this PR do?

Adding Eclipse Dirigible Che Plugin

### CODE REVIEW

**Comment by slemeur** [comment]
Really cool ! thanks @delchev

**Comment by slemeur** [comment]
@benoitf who is going to merge this one?

**Comment by benoitf** [comment]
I'm waiting other approvals then we can merge it

File: plugins/org.eclipse.che.editor.dirigible/1.0.0/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+id: dirigible-che-editor-plugin
```
Comment (skabashnyuk) [inline plugins/org.eclipse.che.editor.dirigible/1.0.0/meta.yaml]:
```suggestion
id: org.eclipse.che.editor.dirigible
```

File: plugins/org.eclipse.che.editor.dirigible/1.0.0/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+version: 1.0.0
+type: Che Plugin
+name: dirigible-che-editor-plugin
+title: Eclipse Dirigible for Eclipse Che
+description: Eclipse Dirigible as App Development Platform for Eclipse Che
+icon: https://www.dirigible.io/img/logo/dirigible-logo.png
+url: https://github.com/dirigiblelabs/dirigible-che-editor-plugin/releases/download/1.0.0/dirigible-che-editor-plugin.tar.gz
```
Comment (skabashnyuk) [inline plugins/org.eclipse.che.editor.dirigible/1.0.0/meta.yaml]:
```suggestion
url: https://github.com/dirigiblelabs/dirigible-che-editor-plugin/releases/download/1.0.0/dirigible-che-editor-plugin.tar.gz

```

File: plugins/org.eclipse.che.editor.dirigible/1.0.0/meta.yaml (hunk 1 above)
Comment (l0rd) [reply plugins/org.eclipse.che.editor.dirigible/1.0.0/meta.yaml]: @skabashnyuk we don't have a naming convention yet. I wouldn't go with the Java packages naming anyway (too verbose). I have created https://github.com/eclipse/che-plugin-registry/issues/55 to discuss that.

Comment (skabashnyuk) [reply plugins/org.eclipse.che.editor.dirigible/1.0.0/meta.yaml]: We have convention that path should be plugins/{id}/{version}/meta.yaml

**Pull request merged**